template-xml/app/src/main/java/co/nimblehq/template/xml/extension/ViewModelExt.kt

```diff
@@ -16,25 +14,13 @@ import androidx.lifecycle.viewmodel.CreationExtras
  * There is another way to resolve the issue as well and it is mentioned in the reference link.
  * Reference: https://proandroiddev.com/testing-the-untestable-the-case-of-the-viewmodel-delegate-975c09160993
  */
-@MainThread
-inline fun <reified VM : ViewModel> Fragment.provideActivityViewModels(
-    noinline extrasProducer: (() -> CreationExtras)? = null,
-    noinline factoryProducer: (() -> ViewModelProvider.Factory)? = null
-): Lazy<VM> = OverridableLazy(activityViewModels(extrasProducer, factoryProducer))
-
 @MainThread
 inline fun <reified VM : ViewModel> Fragment.provideViewModels(
     noinline ownerProducer: () -> ViewModelStoreOwner = { this },
     noinline extrasProducer: (() -> CreationExtras)? = null,
     noinline factoryProducer: (() -> ViewModelProvider.Factory)? = null
 ): Lazy<VM> = OverridableLazy(viewModels(ownerProducer, extrasProducer, factoryProducer))
 
-@MainThread
-inline fun <reified VM : ViewModel> ComponentActivity.provideViewModels(
-    noinline extrasProducer: (() -> CreationExtras)? = null,
-    noinline factoryProducer: (() -> ViewModelProvider.Factory)? = null
-): Lazy<VM> = OverridableLazy(viewModels(extrasProducer, factoryProducer))
-
```

Comment:
@manh-t What do you think of these changes? 

Just want to make sure these can be removed, as they are currently not being used 🙏 

Related to #137 & #139

Reply:
Weird, this comment didn't trigger in the slack channel cc @manh-t

Reply:
@Tuubz @Wadeewee they are the handy functions for unit testing. We didn't have the test for Activity in our template, so these functions are useless. We can remove them IMO (we have a link above for reference if needed ) 🙏

Reply:
@Tuubz @manh-t I do think we should keep them as helper extensions except until we have another approach to do testing with Fragment and Activity. Proguard will clean unused code when doing code obfuscation. 

That's the same as having [common extensions lib](https://github.com/nimblehq/android-common-ktx/tree/develop/common-ktx/src/main/java/co/nimblehq/common/extensions) with no usage yet.

Reply:
@luongvo @Tuubz To make it clean when generating the script for a new project, I don't think we need to keep the unused function. We can add it later in the future if needed, Also @manh-t said 

> We didn't have the test for Activity in our template, so these functions are useless. We can remove them IMO (we have a link above for reference if needed ) 

So, I prefer to remove it for now 🙏